.github/workflows/deploy.yml

```diff
@@ -36,6 +36,11 @@ on:
         description: "Enable Seeding (true, false, chart_value)"
         default: chart_value
         type: string
+      database_deletion:
+        required: false
+        description: "Decide whether Databases should be deleted or not"
+        default: true
```

Comment:
i am not sure what the default value should be: true or false. We could set this to false, to make sure that nothing gets deleted accidently, or make it harder. Then we had to add the database_deletion var in the deploy task in the server, client, keycloak and ldap repo, so that the default in these repo still is, that the Database gets deleted. In case someone does not want the Database destroyed, the could simply comment that line out or set it to false.  What do you think?

Reply:
talkes with Marco, Salima, Michael and Kristoff and default will be false to be sure. Kristoff said that main namespace Database should get deleted regularily but forDev Namespaces there is no need anymore